src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callBack, number) {
+      return callBack(number);
+    },
+
+    add(number) {
+      calculator.result += number;
+
+      return calculator;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+
+      return calculator;
+    },
+
+    divide(number) {
```

Comment:
The 'divide' method could potentially cause a 'divide by zero' error. Please add a check to ensure the 'number' parameter is not zero before performing the division.